src/tck/include/file/params/UpdateFileParams.h

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+#ifndef HIERO_TCK_CPP_UPDATE_FILE_PARAMS_H_
+#define HIERO_TCK_CPP_UPDATE_FILE_PARAMS_H_
+
+#include "common/CommonTransactionParams.h"
+#include "json/JsonUtils.h"
+
+#include <nlohmann/json.hpp>
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace Hiero::TCK::FileService
+{
+/**
+ * Struct to hold the arguments for an `updateFile` JSON-RPC method call.
+ */
+struct UpdateFileParams
+{
+  /**
+   * The ID of the file to update.
+   */
+  std::string mFileId;
```

Comment:
`mFileId` should be `std::optional` (https://hiero-ledger.github.io/hiero-sdk-tck/test-specifications/file-service/FileUpdateTransaction.html#input-parameters)